src/Web/Views/Account/CustomAssessment.cshtml

```diff
@@ -24,21 +24,26 @@
                     <h3>Custom Assessment</h3>
                     <div class="form-group">
                         <div class="col-xs-12">
-                            <div>Use <b>@@deviceContextId</b> in your request to have it replaced with the DFP Fingerprinting session ID.</div>
-                            <div>Use <b>@@deviceIpAddress</b> in your request to have it replaced with the client's IP address.</div>
-                            <pre>
- {
-     ...
-     "deviceContext": {
-         "deviceContextId": "@@deviceContextId",
-         "ipAddress": "@@deviceIpAddress",
-         ...
-     }
-     ...
- }
-                            </pre>
+                            <div class="card">
```

Comment:
tweaked the layout of this a little

![image](https://github.com/microsoft/Dynamics-365-Fraud-Protection-Samples/assets/11251672/8bc29614-bcf0-4070-bb06-ec14b1ce02b8)